requirements.txt

```diff
@@ -1,4 +1,6 @@
-ansible>=2.2
+# ansible 2.2.2.0 inexplicably fails integration tests under tox;
```

Comment:
I did get tests to run fine from a source checkout of ansible-2.2.2.0 but tox uses what pip gets so it seems there is still some difference.